contracts/test-integration/BlsProvider.test.ts

```diff
@@ -411,6 +411,183 @@ describe("BlsProvider", () => {
     expect(chainId).to.equal(expectedChainId);
     expect(accounts).to.deep.equal(expectedAccounts);
   });
+
+  it("should return the number of transactions an address has ever sent", async function () {
+    // Arrange
+    const address = await blsSigner.getAddress();
+    const expectedTransactionCount = await regularProvider.getTransactionCount(
+      address,
+    );
+    const expectedTransactionCountAtEaliestBlockTag =
+      await blsProvider.getTransactionCount(address, "earliest");
```

Comment:
This is the same expression as `transactionCountAtEaliestBlockTag`. Should we be using `regularProvider` here?